README.md

```diff
@@ -1,340 +1,133 @@
-#Objects
+# Introduction to Object Orientation
 
-## What is an object?
+## Objectives
 
-Back in the day, we had procedural programming. 
+1. Understand programming paradigms as a form of human abstraction.
+2. Learn the distinguishing characteristics of Object Oriented Programming (OOP).
+3. Define the terms "methods" and "properties" in the context of OOP.
+4. Use dot notation to access an object's properties.
+5. Chain dot notation to access a property of a property.
 
-######Example (in BASIC)
-```
-READY
-10 PRINT "Hello there."
-20 INPUT "Do you like me? (Type y or n)"; ans$
-30 IF ans$ = "n" then GOTO 50
-40 END "Cool. I like you too!"
-50 PRINT "Maybe you didn't understand the question. Let me ask it again."
-60 GOTO 20
-```
+## Object-Oriented Programming (OOP)
 
-Then, programmers made a transition to naming blocks of code, which became known as functions or methods. All of a sudden instead of going to a specific line number we could say something that was somewhat english-y to tell the program what code we wanted to run next and call it by name.
+*An object-oriented approach to application development makes programs more intuitive to design, faster to develop, more amenable to modification, and easier to understand.*  
+—[*Object-Oriented Programming with Objective-C*][apple_oop_guide_intro], Apple Inc.
 
-######Example (in C)
+[apple_oop_guide_intro]: https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/OOP_ObjC/Introduction/Introduction.html#//apple_ref/doc/uid/TP40005149-CH1-SW2
 
-```
-void DisplayWhoIAm(){
-    printf("I am the writer of this tutorial!");
-}
-```
+Since all computer code boils down to binary it's natural to wonder, "how can a string of ones and zeroes be referred to as an 'object'?" The use of the word "object" is an abstraction of thought. An "object" in code has no more physical form than does a word in any human language. Sure, words have physical representations: speaking a word causes air to vibrate in a sound wave, ink on a page can be shaped into symbols that represent the word, a meaning can be pointed at or mimed out; but none of these are the word itself. Human language is a system of abstraction: it communicates the *idea* of a thing, but not the thing itself.
```

Comment:
remember, they don't know what an "object" in the programming sense is yet! Warm them up a bit to that.